src/ck_play.c

```diff
@@ -1054,7 +1054,8 @@ void CK_CheckKeys()
 		return;
 
 	// Drop down status
-	if (IN_GetKeyState(IN_SC_Enter))
+	// TODO: why does this use IN_GetKeyState() instead of IN_GetLastScan()?
```

Comment:
FYI, this is just because that's what the original game used (it checks its 'Keyboard' array, which is our equivalent of IN_GetKeyState(), rather than checking its 'LastScan' variable). This works properly if there are muliple keys pressed, so maybe that's why(?)